indexer/go.mod

```diff
@@ -2,7 +2,10 @@ module github.com/ethereum-optimism/optimism/indexer
 
 go 1.17
 
-replace github.com/ethereum/go-ethereum v1.10.26 => github.com/ethereum-optimism/op-geth v0.0.0-20230214215134-401b7fd3309b
+replace (
+	github.com/ethereum/go-ethereum v1.10.26 => github.com/ethereum-optimism/op-geth v0.0.0-20230214215134-401b7fd3309b
```

Comment:
In the `require` list below, the version is still at `v1.10.26`. Didn't you update it to `v1.11.2` on purpose?

Reply:
We can change it to `v1.11.2` here in the `replace`, but then it also needs to change to `v1.11.2` in the `require`.

Reply:
I made the comment here because Github doesn't allow me to make comments to non-changed lines ^^ I was referring to the `require` line. We could then remove the redundant replace, it currently mentions two go-ethereum versions.

Reply:
I think we should leave updating the indexer to a separate PR. Indexer wasn't updated to use the new optimism branch with the latest Geth version.  Just switching op-geth to the new version doesn't work as it also lists explicit versions of op-node, op-service, op-e2e and op-bindings which would need to be updated (and then I don't make sense of the various error messages I'm getting and it feels like something too big to tack onto this PR).

Reply:
Thanks for clarifying.